File: two-sum.cpp
class Solution:
    def twoSum(self, nums, target):
        n = len(nums)  # Get the length of the list
        for i in range(n):  # Outer loop to pick the first number
            for j in range(i + 1, n):  # Inner loop to pick the second number
                if nums[i] + nums[j] == target:  # Check if the sum equals the target
                    return [i, j]  # If found, return the indices
        return []  # Return an empty list if no solution is found

File: easy/two-sum.py
class Solution:
    def twoSum(self, nums, target):
        n = len(nums)
        for i in range(n):
            for j in range(i + 1, n):
                if nums[i] + nums[j] == target:
                    return [i, j]
        return []


# Create an instance of the Solution class
if __name__ == "__main__":
    solution = Solution()

    nums = [2, 7, 11, 15]
    target = 9

    result = solution.twoSum(nums, target)
    print(result)
